### PR TITLE
add angular-toastr v1.6.0

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/Foxandxss/angular-toastr
**Watch 40
  Star 700
 Fork 132**
@Amomo could u help me check this PR for #6813 ? thank you!
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update) 
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

**Comment by Foxandxss** [comment]
I am not sure how the auto-update works, but the other checks sound fine for me.

**Comment by maruilian11** [comment]
@Foxandxss thank you!

File: ajax/libs/angular-toastr/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "url": "git+https://github.com/Foxandxss/angular-toastr.git"
+  },
+  "filename": "angular-toastr.min.js",
+  "keywords": [
+    "angularjs",
+    "toastr",
+    "popup"
+  ],
+  "license": "MIT",
+  "description": "[![Code Climate](https://codeclimate.com/github/Foxandxss/angular-toastr.png)](https://codeclimate.com/github/Foxandxss/angular-toastr) [![Build Status](https://travis-ci.org/Foxandxss/angular-toastr.svg?branch=master)](https://travis-ci.org/Foxandxss/ang",
```
Comment (PeterDaveHello) [inline ajax/libs/angular-toastr/package.json]: This description looks strange, should not use markdown here, please update it to a normal form.

Comment (Foxandxss) [inline ajax/libs/angular-toastr/package.json]: Yeah, probably a simple description. You can use something like:

"Angular-toastr is a notification library based on CodeSeven/toastr but written for Angular.js"

Or something like that :)

Comment (PeterDaveHello) [inline ajax/libs/angular-toastr/package.json]: Thanks @Foxandxss

**Comment by maruilian11** [comment]
thank you @Foxandxss 
@PeterDaveHello is that ok now?

**Comment by Foxandxss** [comment]
I released 1.7.0 today. Should that change something for this PR?

**Comment by PeterDaveHello** [comment]
@Foxandxss nope, auto-updater will help me pick it.

**Pull request merged**